Documentation/configuration/metrics.rst

```diff
@@ -29,6 +29,12 @@ Endpoint
 * ``endpoint_regenerating``: Number of endpoints currently regenerating
 * ``endpoint_regenerations``: Count of all endpoint regenerations that have completed, tagged by outcome
 
+Drops/Forwards (L3/L4)
```

Comment:
Are L7 drops separate?

From out-of-band discussion I was under the impression that the forward counts were either incremented from BPF, or from L7 proxy, depending on what kind of policy the traffic is hitting. This title suggests otherwise. Can you clarify?

Reply:
Drop /Forward can be independent of policy in L3-L4. Drop due to policy is simply one of the reasons. The L7 proxy on the other hand will *always* have the policy in place and the drops/forwards will be relevant to policy. Does that make it more clear?

Reply:
@manalibhutiyani From drop perspective I think this makes sense. However I'm still confused from the forwarding count. Let me start a fresh comment thread from the relevant line below.